tgui-next/packages/functional/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @copyright 2018 Aleksej Komarov
+ * @license GPL-2.0-or-later
```

Comment:
Highlighting license for maintainers to review

Reply:
@JJRcop it is compatible with AGPL v3, because of the GPL-2.0-**or-later**. I can remove licensing too, implicitly upgrading it to AGPL v3, but affero's extra clauses are pointless for javascript.

Reply:
>but affero's extra clauses are pointless for javascript.

No they aren't! .min files are considered object files and AGPL forces anybody using them to provide the original version.